src/components/ItaliaTheme/View/Commons/Dates.jsx

```diff
@@ -60,6 +62,9 @@ const Dates = ({ content, show_image, moment: momentlib, rrule }) => {
       forceset: true,
     });
 
+    // overwrite end date variable if event has recurrence
+    actualEndDate = rruleSet.rrules()[0].options.until;
```

Comment:
occhio, potrebbero essere null alcuni di quei campi. Avete per esempio provato a non impostare nulla nei campi della ricorrenza e ad aggiungere ad esempio solo alcune date aggiuntive ?

Reply:
Non è possibile inserire date aggiuntive senza impostare una ricorrenza.
Se si impostano date aggiuntive, in automatico prende i dati della ricorrenza impostati nel widget. Se si impostano date aggiuntive, content.recurrence esiste sempre

Reply:
ok, meglio cosi allora